### DISCUSSION TITLE
How to create newline in Github Bio

### DISCUSSION THREAD

**Discussion by thyoondev** [community/community#18184] [Other Feature Feedback, Questions, & Ideas]
I want to change this

```md
Hello____ 
Thank You______
Bye
```

I want to have newline instead of blanks.

like this

```md
Hello
Thank You
Bye
```

How can i do this?

**Comment by kirillzhosul** [comment]
Try use `\`?
Or even `__` instead of `____` (spaces)?

**Comment by verduscos** [comment]


**Reply by thyoondev** [reply]
not working in github bio.. but thank you

**Reply by BYTESHAUS** [reply]
It's not a solution.

**Comment by alizain-c** [comment]


**Reply by shellscape** [reply]
OP was talking about the _github bio_ and not markdown. `\` nor `<br>` nor any HTML work in the _bio_ field.

**Comment by shellscape** [comment]
The only way to get newlines in the bio field of a github profile is to use unicode whitespace to pad the end of a "line" and force the appearance that a "new line" exists. The Ideographic Space from this chart works well https://jkorpela.fi/chars/spaces.html

**Reply by shologin** [reply]
Yeah, that's works. 
If someone didn't get how to do it (like me after reading this): just go to the link, find the IDEOGRAPHIC SPACE in the table, **copy the "space"** between "foo" and "bar", and paste this **"space"** so much times as you need to move the text to a new line.

**Reply by fbhsdyland2** [reply]
worked for me thanks @shologin

**Reply by JFWooten4** [reply]
POG

**Comment by vrouilhac** [comment]
For those still searching for an answer, checkout my profile, I used this character (between quotes) " ", just copy and paste it.

Note: those takes character so make sure to not fill entire lines with them as the bio's length is limited in characters

**Comment by faizul726** [comment]
### From 2024 here

I was able to make this by using this [↗ dot like symbol](https://pastebin.com/raw/bYb2LP6T) and [↗ IDEOGRAPHIC SPACE](https://pastebin.com/raw/2Czn7E40) from https://jkorpela.fi/chars/spaces.html

i have included `↗ Pastebin links`. Simply go there use Ctrl+A then Ctrl+C

`It took me a while to figure out that I have to spam the space thingy at the end of my sentence until it makes a new line.`

![image](https://github.com/user-attachments/assets/e9bd38bc-296a-4de7-878e-16b2970c7973)
<hr>
You may also try to copy it from here. Not sure if it works...

**Dot like symbol**
```
 ִ 
```
<br>

**IDEOGRAPHIC SPACE**
```
　
```

**Comment by Klegin** [comment]
Hey friens, I tried braille empty pattern unicode + ideographic sapce (it worked for me)

if u want to copy paste, feel free

```
Text Text Text Text Text Text Text.　　　　　　　　　　　
⠀ 　　　　　　　　　　　　　　　　　　　　
More Text.
```

hope it helps ^^
